Patch status: limbo

From alpine's[1]:
  This optimisation of CLONE_SETTLS is not valid used like this, and
  future musl clone(3) will EINVAL on this use.

Have not looked at musl's changes but, given [2], this seems to have
happened at least with >=musl-1.2.5.

[1] https://git.alpinelinux.org/aports/tree/community/qt6-qtwebengine/no-sandbox-settls.patch?id=996cb72e0e48
[2] https://bugs.gentoo.org/937875
--- a/src/3rdparty/chromium/sandbox/linux/services/credentials.cc
+++ b/src/3rdparty/chromium/sandbox/linux/services/credentials.cc
@@ -91,5 +91,5 @@
   void* tls = nullptr;
 #if (defined(ARCH_CPU_X86_64) || defined(ARCH_CPU_ARM_FAMILY)) && \
-    !defined(MEMORY_SANITIZER)
+    !defined(MEMORY_SANITIZER) && defined(__GLIBC__)
   // Use CLONE_VM | CLONE_VFORK as an optimization to avoid copying page tables.
   // Since clone writes to the new child's TLS before returning, we must set a
